packages/react-native/package.json

```diff
@@ -112,7 +112,7 @@
   },
   "dependencies": {
     "@jest/create-cache-key-function": "^29.6.3",
-    "@react-native-mac/virtualized-lists": "workspace:*",
+    "@react-native-mac/virtualized-lists": "0.76.4",
```

Comment:
Can you explain the change from `workspace:*` to `0.76.4`?

Reply:
See the notes in this PR: https://github.com/microsoft/react-native-macos/pull/2318
As Tommy pointed out, I should revert for the main branch.